### PR TITLE
Use Expressive Code component for all code blocks

### PR DESCRIPTION
**Pull request by ncjones**
Ticket Link: https://www.notion.so/immersve/New-Code-Block-Style-fb63f4b3ec9b45bf850adeca24ed9996?pvs=4

### CODE REVIEW

**Review by NicBathgate — DISMISSED** [review]
Awesome 🙌

File: imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-on-chain-card-issuing-integration.mdoc [hunk 1]
```diff
@@ -241,7 +223,6 @@ curl -X POST "https://test.immersve.com/api/spending-prerequisites" \
     "kycRegion":"NZL"
```
Comment (forktrucka) [inline imsv-docs-astro/src/content/docs/guides/card-issuing-apps/custodial-on-chain-card-issuing-integration.mdoc:223]: can we fix this?

File: imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc [hunk 2]
```diff
@@ -59,25 +57,22 @@ reference](/api-reference/get-spending-prerequisites/)
...
+  -H 'Content-Type: application/json' \
+  -H "Authorization: Bearer $token" \
+  --data '{
+    "cardProgramId": "'${card_program_id}'",
+    "fundingSourceId": "'${funding_source_id}'",
+    "spendableAmount": 100,
+    "spendableCurrency": "USD",
+    "kycType": "immersve-conducted",
+    "kycRedirectUrl: "https://app.example.io",
+    "kycRegion": "NZL"
```
Comment (forktrucka) [inline imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc:71]: and this?

File: imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc [hunk 3]
```diff
@@ -124,32 +118,28 @@ allowed to create and use their Immersve card.
...
```
Comment (forktrucka) [inline imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc:132]: here too

**Review by catluin — APPROVED** [review]

**Pull request merged**